assets/src/application/ui/display/Tooltip/types.ts

```diff
@@ -0,0 +1,4 @@
+export interface ClickableIconWithTooltipProps {
+    icon: React.ReactNode;
```

Comment:
Here is that TS issue. You have set it as `ReactNode` rather than `ComponentType`

Reply:
updated there

Reply:
TS is still showing an error in `ClickableIconWithTooltip`, but we will can deal with that when we will move to monorepo, because there will be some changes related to icons